src/main/java/edu/ucsb/cs156/example/controllers/HotelController,java

```diff
@@ -0,0 +1,105 @@
+package edu.ucsb.cs156.example.controllers;
+
+import edu.ucsb.cs156.example.entities.Hotels;
+import edu.ucsb.cs156.example.errors.EntityNotFoundException;
+import edu.ucsb.cs156.example.repositories.HotelsRepository;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.extern.slf4j.Slf4j;
+
```

Comment:
you deleted a line here (not sure this was intentional), missing: 
import com.fasterxml.jackson.core.JsonProcessingException;